.github/workflows/bench-pr.yml

```diff
@@ -218,7 +156,7 @@ jobs:
         run: |
           echo "TMPDIR=/work" >> $GITHUB_ENV
 
-      - name: Run TPC-H benchmark
+      - name: Run benchmark
```

Comment:
you can add `${{ matrix.benchmark.id }}` here

Reply:
Done.